src/test/java/seedu/programmer/model/FileManagerTest.java

```diff
@@ -63,7 +63,10 @@ public void getStudentsFromCsv_wrongNumberOfColumns_throwsIllegalValueException(
 
     @Test
     public void getStudentsFromCsv_invalidField_throwsIllegalArgumentException() {
-        String expectedMessage = "Student ID should only contain 9 alphanumeric characters, and it should not be blank";
+        String expectedMessage = "Student ID should only contain 9 alphanumeric characters, "
+                                + "where the first and last characters "
+                                + " should be alphabets and the rest are numbers. And it should not be blank."
+                                + "\nFor example: A0212345T";
```

Comment:
How about "It should be of the form `AXXXXXXXY` where X is a non-negative integer and Y is the last character of your student ID." 

?

Reply:
sounds good! i'll do that